src/js/components/visualization/ChartMenu.jsx

```diff
@@ -0,0 +1,35 @@
+import changeCase from 'change-case'
+import Download from 'material-ui/svg-icons/file/file-download'
+import MenuItem from 'material-ui/MenuItem/MenuItem'
+import {saveCanvas} from '../../modules/utilities'
+import React, {Component, PropTypes} from 'react'
+
+export class ChartMenu extends Component {
```

Comment:
This should be a functional component since it doesn't use any lifecycle methods